analysis/runtime/src/handlers.rs

```diff
@@ -2,6 +2,11 @@ use crate::events::{Event, EventKind};
 use crate::mir_loc::MirLocId;
 use crate::runtime::global_runtime::RUNTIME;
 
+// WARNING! Most handlers in this file may be called from a signal handler,
+// so they and all their callees should be signal-safe.
```

Comment:
Nit: Isn't the technical term `async-signal-safe` specifically?

```suggestion
// so they and all their callees should be async-signal-safe.
```

Reply:
Right, I'll update the comments. IMHO it's still weird because async-signal-safe implies the existence of sync-signal-safe, which I haven't heard of. Maybe they mean `signalfd`???